File: easy_simple_calculator.py
#!/usr/bin/env python3

a = int(input("Enter the first number: "))
b = int(input("Enter the second number: "))

c = a + b
print(f"{a} + {b} = {c}")

c = a - b
print(f"{a} - {b} = {c}")

c = a * b
print(f"{a} * {b} = {c}")

if a != 0 and b != 0:
    c = a / b
    print(f"{a} / {b} = {c}")
else:
    print("Cannot divide by zero!")

File: easy_simple_calculator.lua
print("Enter the first number: ")
a = tonumber(io.read())
print("Enter the second number: ")
b = tonumber(io.read())

c = a + b
print(a .. " + " .. b .. " = " .. c)

c = a - b
print(a .. " - " .. b .. " = " .. c)

c = a * b
print(a .. " * " .. b .. " = " .. c)

if a ~= 0 and b ~= 0 then
    c = a / b
    print(a .. " / " .. b .. " = " .. c)
else
    print("Cannot divide by zero!")
end